### DISCUSSION TITLE
How to make full page screenshot with com.codeborne:selenide-full-screenshot ?

### DISCUSSION THREAD

**Discussion by VladyslavZagorodnyi** [selenide/selenide#2100] [General]
I need to make a screenshot of my page from header to footer, page is too long to be displayed without scroll, so I need some capabilities to make  full page screenshot. Please suggest something, also provide example if you can. Thanks!

**Comment by asolntsev** [comment]
@VladyslavZagorodnyi You don't need to do anything else but adding dependency `com.codeborne:selenide-full-screenshot` to your project. It will automatically take full-size screenshot. 

P.S. But it takes a screenshot of the content currently rendered in browser. If some content is not rendered yet (some elements will appear only after scrolling down), then it will not be reflected in screenshot. It's not possible to take a picture of something that doesn't exist yet. :)
